features/bevragen/persoon/adressering/aanhef/summary.feature

```diff
@@ -1,16 +1,129 @@
+# language: nl
+
 Functionaliteit: Als gemeente wil ik de juiste en consistente briefaanhef in communicatie naar burgers
   De aanhef is een standaardformulering waar mee de overheid zich tot de geadresseerde richt in de vorm van een introducerende zin in een brief of een email.
```

Comment:
Als consumer van de BRP API
Wil ik de aanhef (introducerende zin van een brief of e-mail) voor een persoon kunnen vragen
Zodat ik de persoon op de juiste en consistente manier kan aanschrijven

zou ik als omschrijving hebben gekozen voor de functionaliteit

Reply:
Aangepast